spec/javascripts/components/projects/project_card_spec.js

```diff
@@ -81,6 +94,19 @@ describe('<ProjectCard />', () => {
         });
       });
 
+      describe('.card-tag', () => {
+        it('has the tag', () => {
```

Comment:
I think the colors should be tested here too, presence and expected value of forecolor.